index.html

```diff
@@ -0,0 +1,102 @@
+                <h2>Search Engine Optimization</h2>
+                <p>
+                    The dominance of mobile internet use means that users are searching for the right business as they travel, shop, or sit on their couch at home. Search Engine Optimization (SEO) allows you to increase your visibility and find the right customers for your
+                    business.
+                </p>
+            </div>
+            <!--Online Reputation Management section-->
+            <div id="online-reputation-management" class="content-section">
+                <img src="./assets/images/online-reputation-management.jpg" class="float-right content-img" alt="Person working on laptop" />
+                <h2>Online Reputation Management</h2>
+                <p>
+                    The web is full of opinions, and some of these can be negative. Social media allows anyone with an internet connection to say whatever they want about your business. Online Reputation Management gives you the control over what potential customers see
+                    when they search for your business.
+                </p>
+            </div>
+            <!--Social Media Marketing section-->
+            <div id="social-media-marketing" class="content-section">
+                <img src="./assets/images/social-media-marketing.jpg" class="float-left content-img" alt="People around desk on media devices, with social media icons on desk" />
+                <h2>Social Media Marketing</h2>
+                <p>
+                    Social media continues to have a sizable influence on buying habits. Social media marketing helps you determine which platforms are suited to your brand, using analytics to find the right markets and increase your lead generation.
+                </p>
+            </div>
+        </article>
+
+        <!--Aside section, will consitst of 3 divs, seperating each benefit-section to contain img, h3 and p on the side of the web page-->
+        <aside class="benefits">
+            <!--Lead Generation section-->
+            <div class="benefit-section">
+                <h3>Lead Generation</h3>
+                <img src="./assets/images/lead-generation.png" alt="Icon for Lead Generation" />
+                <p>
+                    Inbound strategies for lead generation require less work for your business, bringing customers directly to your website.
+                </p>
+            </div>
+            <!--Brand Awareness section-->
+            <div class="benefit-section">
+                <h3>Brand Awareness</h3>
+                <img src="./assets/images/brand-awareness.png" alt="Icon for Brand Awareness" />
+                <p>
+                    Users find your business through paid and organic searches, increasing the search ranking and visibility for your business.
+                </p>
+            </div>
+            <!--Cost Management section-->
+            <div class="benefit-section">
```

Comment:
This `div` tag could of been a `section` tag. This would of made it more semantic.